### PR TITLE
W-11099504 workday reference cleanup

### PR DESCRIPTION
**Pull request by sathya0**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: workday/15.0/modules/ROOT/pages/workday-reference.adoc [hunk 1]
```diff
@@ -250,28 +267,28 @@ a|  The Workday object type that applies to this input source. The following obj
...
 
-== Operations
 
 [[absenceManagement]]
 === Absence Management
 
 `<workday:absence-management>`
 
-Invoke the Absence Management web service. The Absence Management web service contains operations that expose absence-related data, including employee time off and absence inputs for time off and accrual adjustments and overrides, and leave requests, in Workday Human Capital Management business services.
+Invokes the Absence Management web service. The Absence Management web service contains operations that expose absence-related data, including employee time off and absence inputs for time off and accrual adjustments and overrides, and leave requests, in Workday Human Capital Management business services.
```
Comment (dhirsch1) [inline workday/15.0/modules/ROOT/pages/workday-reference.adoc]: The second sentence is large, so it's hard to follow. I would split it in two. Maybe:

The Absence Management web service contains operations that expose absence-related data in Workday Human Capital Management  business service.  This data includes employee time off, absence inputs for time off, accrual adjustments and overrides for time off, and leave requests.

File: workday/15.0/modules/ROOT/pages/workday-reference.adoc [hunk 2]
```diff
@@ -2706,252 +2754,327 @@ Invoke the Workforce Planning web service, which contains operations that expose
...
 | Field | Type | Description | Default Value | Required
-| Only End Entities a| Boolean | Verify the last element of the certificate chain only. |  |
-| Prefer Crls a| Boolean | Try CRL instead of OCSP first. |  |
-| No Fallback a| Boolean | Do not use the secondary checking method, which is the one selected in the Prefer Crls field. |  |
-| Soft Fail a| Boolean | Avoid verification failure when the revocation server cannot be reached or is busy. |  |
+| Host a| String | Hostname or IP address of the proxy server. |  | x
+| Port a| Number | Port of the proxy server. |  | x
+| Username a| String | Username to authenticate against the proxy server. |  |
+| Password a| String | Password to authenticate against the proxy server. |  |
+| Non Proxy Hosts a| String | Non proxy hosts. |  |
```
Comment (dhirsch1) [inline workday/15.0/modules/ROOT/pages/workday-reference.adoc]:
```suggestion
| Non Proxy Hosts a| String | Non-proxy hosts. |  |
```

File: workday/15.0/modules/ROOT/pages/workday-reference.adoc [hunk 3]
```diff
@@ -2706,252 +2754,327 @@ Invoke the Workforce Planning web service, which contains operations that expose
...
+| Password a| String | Password used to protect the keystore. |  |
+| Algorithm a| String | Encryption algorithm that the keystore uses. |  |
 |===
 
-[[Reconnection]]
-=== Reconnection
+[[non-repeatable-stream]]
+=== Non-Repeatable Stream
+
+Disable the repeatable stream functionality and use non-repeatable streams to have less performance overhead, memory use, and cost.
```
Comment (dhirsch1) [inline workday/15.0/modules/ROOT/pages/workday-reference.adoc]:
```suggestion
Disables the repeatable stream functionality and uses non-repeatable streams to have lower performance overhead, memory use, and cost.
```

File: workday/15.0/modules/ROOT/pages/workday-reference.adoc [hunk 4]
```diff
@@ -2706,252 +2754,327 @@ Invoke the Workforce Planning web service, which contains operations that expose
...
+Configures the in-memory streaming strategy by which the request fails if the data exceeds the MAX buffer size. Always run performance tests to find the optimal buffer size for your specific use case.
 
-[%header%autowidth.spread]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| In Memory Size a| Number | Defines the maximum memory that the stream should use to keep data in memory. If more than that is consumed, then it will start to buffer the content on disk. |  |
+| Initial Buffer Size a| Number | Initial amount of memory to allocate to the data stream. If the streamed data exceeds this value, the buffer expands by *Buffer Size Increment*, with an upper limit of *Max In Memory Size* value. |  |
+| Buffer Size Increment a| Number | Amount by which the buffer size expands if it exceeds its initial size. Setting a value of `0` or lower specifies that the buffer can't expand. |  |
+| Max Buffer Size a| Number | Maximum amount of memory to use. If more than that is used then a `STREAM_MAXIMUM_SIZE_EXCEEDED` error is raised. A value lower than or equal to zero means no limit. |  |
```
Comment (dhirsch1) [inline workday/15.0/modules/ROOT/pages/workday-reference.adoc]: Global for this field

Description from the reference spreadsheet:

Maximum size of the buffer. If the buffer size exceeds this value, Mule raises a `STREAM_MAXIMUM_SIZE_EXCEEDED` error. A value of less than or equal to `0` means no limit.

File: workday/15.0/modules/ROOT/pages/workday-reference.adoc [hunk 5]
```diff
@@ -2706,252 +2754,327 @@ Invoke the Workforce Planning web service, which contains operations that expose
...
-[%header%autowidth.spread]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Host a| String |  |  | x
-| Port a| Number |  |  | x
-| Username a| String |  |  |
-| Password a| String |  |  |
-| Non Proxy Hosts a| String |  |  |
+| Transport Additional Data a| Object | Transport additional data. |  |
```
Comment (dhirsch1) [inline workday/15.0/modules/ROOT/pages/workday-reference.adoc:2995]: The descriptions in this row and the next aren't informative at all. I suggest noting this in a readme for this connector so we can try to get better ones in the future.

**Review by dhirsch1 — DISMISSED** [review]

File: workday/15.0/modules/ROOT/pages/workday-reference.adoc (hunk 1 above)
Comment (sathya0) [reply workday/15.0/modules/ROOT/pages/workday-reference.adoc]: much better!

**Review by dhirsch1 — APPROVED** [review]
